sessions/slides/introduction-to-ensembles.qmd

```diff
@@ -99,4 +106,8 @@ Many uncertainties, many approaches: many models
 
 [Return to the session](../forecast-ensembles)
 
+## References
+
+::: {#refs}
+:::
```

Comment:
I think if we add the markdown ref it might work as a link in which case we don't need this - but I don't mind having it.